index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;900&display=swap"
+        rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Montserrat:wght@900&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+
+    <nav class="nav-bar">
+        <div class="name">Jorge Rios</div>
+        <ul class="list">
+            <li class="item"><a href="#">Portfolio</a></li>
+            <li class="item"><a href="#">About</a></li>
+            <li class="item"><a href="#">Contact</a></li>
+            <li class="social">
+                <a href="#"><img src="images/mail.svg" alt="email icon"></a>
+            </li>
+        </ul>
+        <div class="menu">
+            <div class="line"></div>
+            <div class="line"></div>
+            <div class="line"></div>
+        </div>
+    </nav>
+    <section class="headline">
+        <div class="headline-icons">
+            <img src="images/github.svg" alt="github icon">
+            <img src="images/linkedinicon.svg" alt="linkedin icon">
+            <img src="images/angellist.svg" alt="angel icon">
+            <img src="images/twitter.svg" alt="twitter icon">
+            <img src="images/medium.svg" alt="medium icon">
+        </div>
+        <div class="headline-description">
+            <h1>Hey There. I’m Jorge</h1>
+            <h4>I’am a Software Developer</h4>
+            <p>I can help you build a product , feature or website Look through some of my work and experience! If you
+                like what you see and have a project you need coded,
+                don’t hestiate to contact me.</p>
+        </div>
+        <div class="headline-ilustration">
+            <img src="images/Illustration-header.svg" alt="ilustration image">
+        </div>
+    </section>
```

Comment:
The general design is very good! All background images, margins, and details are really similar to Figma's! Awesome job